integration/openshift/e2e/gen_test_cases.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script should be used to generate the list of test cases from
+# the OpenShift e2e conformance suite.
+#
+# Edit the filter_patterns, exclude_patterns, and skip_patterns variables
+# and re-run this script in order to change the tests selection.
+#
+set -e
+
+GOPATH=${GOPATH:-$(go env GOPATH)}
+script_dir="$(realpath $(dirname $0))"
+source "$script_dir/../../../lib/common.bash"
+tests_dir="$script_dir/../../../"
+source "$script_dir/../lib.sh"
+
+# Tests that should be included. Leave it empty to pick them all.
+filter_patterns=(
+        )
+
+# Tests that should be excluded.
+exclude_patterns=(
+        )
+
+# Tests that would normally be executed but due to some issue they should not.
+skip_patterns=(
+        )
+
+# Define temporary files and write down the patterns. Don't delete the files
+# afterwards because they can be used to debug.
+#
+inclusion_file="${script_dir}/tests_inclusion_patterns"
+exclusion_file="${script_dir}/tests_exclusion_patterns"
+skip_file="${script_dir}/tests_skip_patterns"
+tmp_tests_file="${script_dir}/new_test_cases"
+test_cases_file="${script_dir}/test_cases.txt"
+printf "%s\n" "${filter_patterns[@]}" > "${inclusion_file}"
+printf "%s\n" "${exclude_patterns[@]}" > "${exclusion_file}"
+printf "%s\n" "${skip_patterns[@]}" > "${skip_file}"
+
+# The OpenShift e2e suite used to generate our list of tests.
+suite="openshift/conformance/parallel"
+
+function print_header()
+{
+cat <<EOF
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Auto-generated by $(basename $0) ($(date '+%Y-%m-%d'))
+#
+EOF
+}
+
+openshift_tests=$(build_openshift_tests) || \
+	die "Failed to build openshift-tests"
+
+${openshift_tests} run --dry-run $suite 2>/dev/null | \
+	grep -E -f ${inclusion_file} > $tmp_tests_file
+
+if [ ${#exclusion_patterns[@]} -gt 0 ]; then
+	cat $tmp_tests_file | \
+		grep -E -v -f ${exclusion_file} >> $tmp_tests_file
+fi
+
+if [ ${#skip_patterns[@]} -gt 0 ]; then
+	cat $tmp_tests_file | \
+		grep -v -f ${skip_file} >> $tmp_tests_file
```

Comment:
Any reason to use use extended regexps in one case and not the other?

Reply:
Yes, the reason is: I did a mistake. :) Gonna fix that.

Reply:
> Yes, the reason is: I did a mistake. :) Gonna fix that.

That sounds like a good reason ;-)